allure-report-face/src/components/widgets-grid/WidgetsGridView.js

```diff
@@ -8,7 +8,7 @@ import allurePlugins from '../../pluginApi';
 
 const widgetTpl = (id) => `<div class="widget island" data-id="${id}">
     <div class="widget__handle">
-        <span class="fa fa-ellipsis-v"></span> <span class="fa fa-ellipsis-v"></span>
+       <span class="draggable-icon"></span>
```

Comment:
1. Icon can be added as background to `.widget__handle`. Stick on BEM.
2. You are not solving issue of implicit draggable widgets at all. This icon also not obvious.